vhdl_parser/src/tokenstream.rs

```diff
@@ -76,7 +76,7 @@ impl TokenStream {
         Ok(self.peek()?.map(|ref token| token.kind))
     }
 
-    pub fn is_peek_kinds(self: &mut Self, kinds: &[Kind]) -> ParseResult<bool> {
+    pub fn get_peek_kinds(self: &mut Self, kinds: &[Kind]) -> ParseResult<bool> {
```

Comment:
I do not think `get_` is a good prefix for this function. Maybe rename it to `next_kinds_are`. It is just a check if the next sequence of tokens in the stream match the sequence of kinds in the argument. It is not *getting* anything.

Reply:
054e4ec